### PR TITLE
fix: [MeshMap] Issues with spacing in "Save time with patterns" section

### LINKED ISSUE #3693: [MeshMap] Issues with spacing in "Save time with patterns" section
A few padding / spacing issues in mobile view:

Screenshot 2023-02-03 at 3 32 53 PM

The line height / padding between lines is a bit too much in small to medium window sizes. Please reduce the whitespace.

Screenshot 2023-02-03 at 3 34 52 PM

_Originally posted by @leecalcote in https://github.com/layer5io/layer5/issues/3690#issuecomment-1416427185_

### PR DESCRIPTION
**Pull request by ritiksaxena124**
**Description**
- removed extra divs
- changed some display property
- responsive media queries added
 
This PR fixes #3693

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Deploy preview building...

**Comment by l5io** [comment]
🚀 Preview for commit 3a75183e2671505851867c46610eb2cc3040263f at: https://63e807f43d8d357a87e7eeaa--layer5.netlify.app

**Review by Shivam-AfA — COMMENTED** [review]
image

Some alignment issues here, rest seems good.

**Comment by l5io** [comment]
🚀 Preview for commit 9c96f8b848516f16747d5c7195f41dbcc23dd780 at: https://63ea7978ceb46e0e7173d437--layer5.netlify.app

**Review by Shivam-AfA — COMMENTED** [review]
@ritiksaxena124 Good work here. Good of you to drop (Row, Column) and use Grid. There's still some spacing issues. 

Currently it looks like this.
image

Whereas it should look like this.
image

That is, equal spacing among the icons, the vertical line and the text.

**Comment by leecalcote** [comment]
Re-running build...

**Comment by l5io** [comment]
🚀 Preview for commit 692d48a0d086c4231575fb3b09b34bc8254beade at: https://63ed20d947e07a391dbb601e--layer5.netlify.app

File: src/sections/Meshmap/meshmap-catalog.js [hunk 1]
```diff
@@ -106,81 +120,59 @@ h2.heading {
   font-family: 'Qanelas Soft','Qanelas Soft', sans-serif;
   font-style: normal;
   font-weight: 50;
-  font-size: 70px;
-  line-height: 88px;
+  font-size: 3.75rem;
   text-align: right;
-  margin-bottom: 4rem;
-
+  margin: 0 0 4rem 0;
```
Comment (Shivam-AfA) [inline src/sections/Meshmap/meshmap-catalog.js:125]: `margin: 2rem 0 4rem`

And we'll be ready to merge.

**Comment by l5io** [comment]
🚀 Preview for commit 3cb1fd1d2a752c1ab018c2d57c8bdaa32bf21633 at: https://63ef96990508010d256cca1b--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**